app/javascript/app/components/scroll-to-highlight-index/scroll-to-highlight-index.js

```diff
@@ -19,23 +20,8 @@ class ScrollToHighlightIndex extends PureComponent {
       ? document.querySelectorAll(targetElementsSelector)[idx]
       : document.querySelectorAll(targetElementsSelector)[0];
 
-    const topPosition = element => {
-      const PADDING = -200;
-      let firstLevelElement = element;
-      let elementPosition = firstLevelElement.offsetTop;
-      while (firstLevelElement.id !== 'ndc-content-container') {
-        elementPosition += firstLevelElement.offsetParent.offsetTop;
-        firstLevelElement = firstLevelElement.offsetParent;
-      }
-      return elementPosition + PADDING;
-    };
-
     if (e) {
-      window.scrollTo({
-        behavior: 'smooth',
-        left: 0,
-        top: topPosition(e)
-      });
+      scrollIt(document.querySelector(targetElementsSelector), 300, 'smooth');
```

Comment:
We have an index that you are missing so I think this won't work well because we are scrolling to the first element, isn't `e` instead?